src/components/table/OcTable.spec.js

```diff
@@ -92,7 +91,6 @@ describe("OcTable", () => {
     })
 
     expect(wrapper.findAll("oc-thead-stub").length).toBe(0)
-    expect(wrapper).toMatchSnapshot()
```

Comment:
Screw 💩  code snapshots... the cons heavily outweigh the pros of using them... saying this as the guy who actually implemented the first one in this codebase 🙈

Reply:
They have saved me more than once from committing crap so 🤷🏽 kudos to the original implementator!